code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -436,13 +436,19 @@
 	pixel_x = -64
 	pixel_y = -64
 	unacidable = 1
-	var/mob/living/immune = null // the one who creates the timestop is immune
+	var/mob/living/immune = list() // the one who creates the timestop is immune
 	var/freezerange = 2
 	var/duration = 140
 	alpha = 125
 
 /obj/effect/timestop/New()
 	..()
+	for(var/mob/M in player_list)
```

Comment:
Should this not be a part /timestop/wizard? Don't think having the timestop spell should stop ALL instances of timestop, just other instances of the spell. Unless this is what you want.